### DISCUSSION TITLE
Send data with esp32

### DISCUSSION THREAD

**Discussion by saeidimajid** [Mair/esp32-course#197] [Course Qeustions]
Hi everyone

How can I send sensor's data to another network with esp32?
for example sensor is in germany and i want to send sensor's data with esp32 to another country.
and how can I get data?

thank's

**Comment by Mair** [comment]
The simplest would most likely be MQTT. Otherwise you'll need to expose a port on a static IP oe use dynamic DNS. This is all doable but gets a little complicated. The simplest is to use MQTT

**Comment by saeidimajid** [comment]
What about using web on chip?
Can I access the esp32 ip from another country to see the sensor values on the chip webpage?

**Comment by Mair** [comment]
Yes this is possible. If you are hosting the web on the chip you will need port forward to a static IP or you can use dynamic DNS.
this subject gets quite involved. let me know if you are interested in the details
